### ISSUE TITLE
Old .flr files support

### ISSUE THREAD

**Issue by kirimi** [ronba/vscode-rive-viewer#2]
Hi!  Thank you for this extension!
Do you have any plans to support old .flr files? This is an old Flare format.

**Comment by ronba** [comment]
Hey Kirimi - apologies for missing this one.
No - no real plans to be honest. I haven't really worked with flr before and the current project I'm working on is only using the new format.

**Comment by kirimi** [comment]
Yes, this is not particularly needed at the present time. All new projects are already on rive.
I just have an old one with a lot of .flr files. But I can handle it manually. )

Thank you and good luck!
